qiskit/algorithms/state_fidelities/compute_uncompute.py

```diff
@@ -49,10 +49,25 @@ class ComputeUncompute(BaseStateFidelity):
 
     """
 
-    def __init__(self, sampler: BaseSampler, options: Options | None = None) -> None:
-        """
+    def __init__(
+        self,
+        sampler: BaseSampler,
+        average_local: bool = False,
```

Comment:
Could we just call this `local`? The "average" part sounds a bit confusing to me and I think in literature it's just referred to as "local fidelity" 🤔